theano/sparse/basic.py

```diff
@@ -233,9 +233,7 @@ def conv_op(*inputs):
         out = op(*ipt)
         return oconv(out)
 
-    seed_rng()
-    rng = numpy.random
-    T.verify_grad(conv_op, dpt, 2, rng, *args, **kwargs)
```

Comment:
It is tests that is calling verify_grad and sparse_verify_grad. The
seed_rng seed the default numpy rng.

On Wed, Sep 7, 2016 at 9:58 AM, kuza55 <notifications@github.com> wrote:

> In theano/sparse/basic.py
> <https://github.com/Theano/Theano/pull/4943#discussion_r77826824>:
>
> > @@ -233,9 +233,7 @@ def conv_op(*inputs):
> >          out = op(*ipt)
> >          return oconv(out)
> >
> > -    seed_rng()
> > -    rng = numpy.random
> > -    T.verify_grad(conv_op, dpt, 2, rng, *args, **kwargs)
>
> Is the seed_rng() actually doing anything though? It seems that it's
> fetching a seed from config.unittests.rseed, but is that actually non-None
> when someone calls verify_grad_sparse? I'm not clear who the expected
> callers for this function are.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4943/files/3f68a471179587a0b14b8eceab544407806712be..78df8f341f7daba5ceb18d4956e5efb2bf00921f#r77826824>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALC-wQxUFk64gqZnALtMWJYxxh1UFTAks5qnsMIgaJpZM4J2a26>
> .
>